sklearn/pipeline.py

```diff
@@ -1578,15 +1578,24 @@ def fit(self, X, y=None, **fit_params):
             Targets for supervised learning.
 
         **fit_params : dict, default=None
-            Parameters to pass to the fit method of the estimator.
+            - If `enable_metadata_routing=False` (default):
+                Parameters directly passed to the `fit` methods of the
+                transformers.
+
+            - If `enable_metadata_routing=True`:
+                Parameters safely routed to the `fit` methods of the
+                transformers. See :ref:`Metadata Routing User Guide
+                <metadata_routing>` for more details.
+
+                .. versionchanged:: 1.5
```

Comment:
we should always say what changed. For instance, in another estimator we say:

```
                    `**partial_fit_params` are routed to the sub-estimator, if
                    `enable_metadata_routing=True` is set via
                    :func:`~sklearn.set_config`, which allows for aliasing.
```

Reply:
Hm, as a versionchanged comment, I would expect some info about what exactly has changed. Not information about the added functionality.

What about:
```
.. versionchanged:: 1.5
`**fit_params` can be routed via metadata routing API
```